cmd/acraserver/config.go

```diff
@@ -41,6 +43,18 @@ type Config struct {
 	ConnectionWrapper       network.ConnectionWrapper
 }
 
+type ConfigProxy struct {
```

Comment:
I was confused when read `ConfigProxy` in `acraserver` package because now acraserver shouldn't know about acraproxy and can operate with clients directly. But when read field's name I understood that you mean proxying arcaserver's config to webui config. Maybe better to rename to `UIEditableConfig` or something more understandable?

Reply:
accepted